src/acl_hostch.cpp

```diff
@@ -834,6 +949,12 @@ void acl_write_program_hostpipe(void *user_data, acl_device_op_t *op) {
     return;
   }
 
+  // Difference between event->cmd.info.host_pipe_info
```

Comment:
Thanks Sophie for the suggestion. I will change the name of the `event->hostpipe_info` to the dynamic info. And I want to keep the host_pipe_info unchanged since it's the same struct used by the legacy code as well, and I don't want to change the name of all of that.

I hope `host_pipe_dynamic_info` and `host_pipe_info` is clear for the difference.